### PR TITLE
landing ui changes

### PR DESCRIPTION
**Pull request by shilohgreen**
all UI changes made in accordance with this [ticket](https://www.notion.so/switcheo/Demex-7d3938451d6d4d899cc814181c78e39e?p=8be1fa905edf460496bcaf6a56a5bd46&pm=s)

### CODE REVIEW

**Comment by andrewsoon** [comment]
image

at width 560px alignment should still be centered

**Comment by andrewsoon** [comment]
image

font size of header on tablet is larger than on laptop

File: src/@demex-info/views/HeroSection/components/SocialsBar.tsx [hunk 1]
```diff
@@ -16,46 +16,46 @@ const SocialsBar: React.FC = () => {
...
-      justifyContent: "space-around",
-      backgroundColor: theme.palette.background.tertiary,
-      boxShadow: `0px 0px 16px ${theme.palette.shadow}`,
-      backdropFilter: "blur(2px)",
-      padding: "0 2rem",
-		[theme.breakpoints.down("sm")]: {
+    position: "fixed",
+    zIndex: 999,
+    width: "265px",
+    height: "55px",
```
Comment (andrewsoon) [inline src/@demex-info/views/HeroSection/components/SocialsBar.tsx:45]: remove height and set padding accordingly, right now the left right padding doesnt equal to the top and bottom spacing

Comment (shilohgreen) [reply src/@demex-info/views/HeroSection/components/SocialsBar.tsx]: I've changed the way the socials bar is written component from flex box to spacing using padding, do let me know if there's any issues!

File: src/@demex-info/views/HeroSection/components/SocialsBar.tsx [hunk 2]
```diff
@@ -16,52 +16,50 @@ const SocialsBar: React.FC = () => {
...
+    [theme.breakpoints.down("sm")]: {
       left: "50%",
       right: 0,
       transform: "translate(-50%, 0)",
     },
   },
   iconBtn: {
+    "&:not(:last-child)": {
+      paddingRight: "1rem",
+    },
```
Comment (andrewsoon) [inline src/@demex-info/views/HeroSection/components/SocialsBar.tsx:62]: probably dont need this

File: src/@demex-info/views/HeroSection/components/SocialsBar.tsx [hunk 3]
```diff
@@ -16,52 +16,50 @@ const SocialsBar: React.FC = () => {
...
+    </Box>
+  );
 };
 
 
 const useStyles = makeStyles((theme) => ({
   root: {
-      position: "fixed",
-      zIndex: 999,
-      width: "265px",
```
Comment (andrewsoon) [inline src/@demex-info/views/HeroSection/components/SocialsBar.tsx:44]: think can keep the width

**Review by andrewsoon — APPROVED** [review]

**Pull request merged**